homeassistant/components/atome/sensor.py

```diff
@@ -247,6 +278,7 @@ def update(self):
             }
         else:
             self._attr_state = getattr(self._data, f"{self._sensor_type}_usage")
+            self._attr_last_reset = dt_util.parse_datetime(getattr(self._data, f"{self._sensor_type}_last_reset"))
```

Comment:
Does this return a UTC time?

Reply:
It returns a date time with timezone.
For instance it returns `2021-08-08T00:00:00+02:00` which is correct for France where it runs

Reply:
Ok. Times in the state machine must always be UTC, so we need to convert this to UTC time. We can use this util:
https://github.com/home-assistant/core/blob/02459e68132b255b0b8a069e88ff022d1eae3ee2/homeassistant/util/dt.py#L69